src/hooks/useLocalStorage.ts

```diff
@@ -0,0 +1,57 @@
+import { useState, useRef } from 'react';
+
+declare global {
+  interface Window {
```

Comment:
wondering if this should go to `src/types` as we might want to define more Window items later?

Reply:
Well, the types are not correct. It's just to make typescript happy in this file with the fake polyfill. So I don't know

Reply:
Oh I missed that. So if it's not there it just runs the function instead 👍

Reply:
Ok.